src/System.Runtime/ref/System.Runtime.cs

```diff
@@ -3675,6 +3675,56 @@ public sealed partial class WeakReference<T> : System.Runtime.Serialization.ISer
         public void SetTarget(T target) { }
         public bool TryGetTarget(out T target) { throw null; }
     }
+
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Auto)]
```

Comment:
Why is `StructLayout` specified here in the ref but not the implementation? Is it even necessary?

Reply:
I have no idea. I had a look at the rest of the file and followed suit. Looking again it seems as though the vast majority include it.